components/Common/Card/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useMemo, type ComponentProps, type FC } from 'react';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Preview from '@/components/Common/Preview';
+import { shortHumanReadableDate } from '@/util/shortHumanReadableDate';
+
+import styles from './index.module.css';
+
+type Author = {
+  firstName: string;
+  lastName: string;
+  src: ComponentProps<typeof AvatarGroup>['avatars'][number]['src'];
+};
+
+export type CardProps = {
+  title: ComponentProps<typeof Preview>['title'];
+  type: ComponentProps<typeof Preview>['type'];
+  subtitle: string;
+  description: string;
+  author: Author;
+  date: Date;
+};
+
+const Card: FC<CardProps> = ({
+  title,
+  type,
+  subtitle,
+  description,
+  author,
+  date,
+}) => {
+  const avatars = useMemo(() => {
+    const avatars = [
+      {
+        alt: `${author.firstName} ${author.lastName}`,
+        src: author.src,
+      },
+    ];
+
+    return avatars;
+  }, [author.firstName, author.lastName, author.src]);
+
+  return (
+    <article className={styles.container}>
+      <Preview
+        title={title}
+        type={type}
+        height="auto"
+        className={styles.preview}
+      />
+      <p className={styles.subtitle}>{subtitle}</p>
+      <h3 className={styles.title}>{title}</h3>
+      <p className={styles.description}>{description}</p>
+      <footer className={styles.footer}>
+        <AvatarGroup avatars={avatars} />
+        <div>
+          <p className={styles.author}>
+            {author.firstName} {author.lastName}
+          </p>
+          <p className={styles.date}>{shortHumanReadableDate(date)}</p>
```

Comment:
Great, thank you! I was not sure about using it, as I tend to use it for dates that are not fully represented, which is the case for all the examples they have on MDN:

```html
<time datetime="2018-07-07">July 7</time>
<time datetime="20:00">20:00</time>
<time datetime="PT2H30M">2h 30m</time>
```

It will still be useful for screen readers, though, so I will definitely add it!